sources/packages/backend/apps/api/src/route-controllers/application-restriction-bypass/models/application-restriction-bypass.dto.ts

```diff
@@ -15,3 +21,41 @@ export class ApplicationRestrictionBypassSummary {
 export class ApplicationRestrictionBypassHistoryAPIOutDTO {
   bypasses: ApplicationRestrictionBypassSummary[];
 }
+
+export class BypassRestrictionAPIInDTO {
+  @IsPositive()
+  applicationId: number;
+  @IsPositive()
+  studentRestrictionId: number;
+  @IsEnum(RestrictionBypassBehaviors)
+  bypassBehavior: RestrictionBypassBehaviors;
+  @IsString()
```

Comment:
Will it ensure that a string different than `""` is not provided?
Usually we use the `@IsNotEmpty()`.